docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
> Do you have a larger mailbox? 

Yes

> You could just grep through the mails, if there are many with spamassassin-stuff from other mail hosts. I will try that as well.

I have messages with these headers dating back to 2006, but I can't know for sure whether or not it was my server that added them.

I'm just thinking if we're removing these headers, why not remove more.

On the other hand, perhaps we shouldn't be removing them after all, since preserving them might help with debugging etc. The important thing is not to consider them authoritative. I just got an e-mail from LibreOffice mailing list, and it contains all these X-Spam headers from LibO server.

The only downside of keeping them I can think of right now is that they may make the decision more difficult for MUA's which can be told to treat them as authoritative. If one `X-Spam-Flag` says "NO", and the other says "YES", which one is the right one?
On the other hand, mail server admin might be about the only person with a "my server knows best" opinion, everyone else might be more open or not care enough.

I'm now in favor of at least commenting out removal of these headers by default. :)